kythe/go/platform/tools/kzip/viewcmd/viewcmd.go

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2019 The Kythe Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package viewcmd displays the contents of compilation units stored in .kzip files.
+package viewcmd
+
+import (
+	"context"
+	"fmt"
+	"flag"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+	"log"
+	"github.com/golang/protobuf/proto"
+	"golang.org/x/sync/errgroup"
+	"golang.org/x/sync/semaphore"
+	"github.com/google/subcommands"
+	"kythe.io/kythe/go/platform/kzip"
+	"kythe.io/kythe/go/util/cmdutil"
+	"github.com/golang/protobuf/jsonpb"
+
+	// These are common detail messages used by Kythe compilations, and
+	// required for JSON (un)marshaling to work.
+	_ "kythe.io/kythe/proto/buildinfo_go_proto"
+	_ "kythe.io/kythe/proto/cxx_go_proto"
+	_ "kythe.io/kythe/proto/go_go_proto"
+	_ "kythe.io/kythe/proto/java_go_proto"
```

Comment:
nit: putting these in a separate file in the same package, which just does the imports might make it easier to change these on import.